BOJ/1330/hikang.py

```diff
@@ -0,0 +1,9 @@
+A,B = map(int,input().split())
+
+# 수 비교해서 알맞은 부등호 출력
+if A > B:
+    print('>')
+elif A < B:
+    print('<')
+else:
+    print('==')
```

Comment:
```suggestion
print('==' if A == B else '<' if A < B else '>')
```
파이썬은 이런게 됨

그래도 보통 저렇게 까진 안하고
`행동a` if `조건` else `행동b` 이정도는 많이 써 (조건 1개에 따라 행동 2가지로 나뉠 때)

Reply:
옛서!~어제 밤에 비몽사몽한 상태로 끝냇다,,ㅎㅎ논문이랑 일 좀 끝나면 나도 코멘트 좀 써야겟어 신경을 못쓰네 여기에,ㅎㅎ